source/CardGames.py

```diff
@@ -104,6 +104,15 @@ def clearHand(self):
     self.hand = []
     self.knownCards = []
 
+  def highCard(self):
```

Comment:
Everything looks great here! Code is clean and concise.

I think everyone has agreed as well to include comments next sprint so we can easily see what we are doing in each function. Great job Joe!

Reply:
Sounds good. I will add that to my code in the future.